forms/aws/lambda.tf

```diff
@@ -210,7 +210,8 @@ resource "aws_lambda_function" "templates" {
       REGION    = var.region,
       DB_ARN    = aws_rds_cluster.forms.arn,
       DB_SECRET = aws_secretsmanager_secret_version.database_secret.arn,
-      DB_NAME   = var.rds_db_name
+      DB_NAME   = var.rds_db_name,
+      TOKEN_SECRET = aws_secretsmanager_secret_version.token_secret.arn
```

Comment:
This is failing with a format error.  Running `terraform fmt` in the `forms/aws` folder will fix it up for you.